test/library/standard/IO/doc-examples/FormattedIO/example_formatted_io_1.chpl

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
nit: What would it take to collapse these files `_1..5` into e.g., `example_formatted_io.chpl` and `example_formatted_io_regex.chpl`? Just to reduce the amount of file clutter in the directory.